### PR TITLE
Replace current red-text highlight with a full background-color highlight

### PR DESCRIPTION
**Pull request by astorije**
~⚠️ **This PR is currently set after my branch at #2315 so _Do Not Merge_ until that other PR makes it in, but making it its own PR to facilitate the review.**~
*All good now.*

---

Supersedes #1847 and #1846.

Inspired by @xPaw's approach at https://github.com/thelounge/thelounge/pull/1847#issuecomment-381578238.
I am using colors given by https://color.adobe.com/ to match our main color to match it with our actual "new" blue.

**Default theme:**

screen shot 2018-06-04 at 00 37 44

**Morning theme:**

screen shot 2018-06-04 at 00 44 08

### CODE REVIEW

**Comment by xPaw** [comment]
Is it actually blocked on that PR? The commit that changes highlights doesn't seem to conflict. You just brought in 2 extra commits that are unrelated.

**Comment by astorije** [comment]
> Is it actually blocked on that PR? The commit that changes highlights doesn't seem to conflict.

Yeah I get a few conflicts that I can't be bothered to fix. I'm hoping we can be done with #2315 in a couple days so this one can make it then.

> You just brought in 2 extra commits that are unrelated.

That's because I rebase like an idiot. Fixed.

File: client/css/style.css [hunk 1]
```diff
@@ -1365,11 +1365,27 @@ background on hover (unless active) */
...
-#chat .error .from,
-#chat .channel .highlight .from,
-#chat .channel .highlight .text,
-#chat .channel .highlight .user {
-	color: #f00;
+#chat .error .from {
+	color: #e74c3c;
+}
+
+#chat .message.highlight {
```
Comment (astorije) [inline client/css/style.css]: I realized that I accidentally removed `.channel` and therefore this highlights in queries too. I was about to re-add it, then wondered if `.highlight` class should be in queries at all. Any insight, @xPaw?

**Comment by astorije** [comment]
@xPaw, fixed the highlights in queries, and removed the paddings on mobile. Should be good to go.

File: client/css/style.css [hunk 2]
```diff
@@ -1365,11 +1369,27 @@ background on hover (unless active) */
...
+	width: 50px;
+	color: #696969;
+}
+
+#chat .channel .message.highlight .content {
+	border-left: 1px solid var(--highlight-bg-color);
+}
+
+#chat .channel .message.highlight .from::after {
+	background: transparent;
```
Comment (xPaw) [inline client/css/style.css]: What does this do?

Comment (astorije) [reply client/css/style.css]: Removed. It was a relic from https://github.com/thelounge/thelounge/pull/2270/files#diff-97db1f70168fb5f12457b238ff6052b5L2230, thanks for noticing!

**Pull request merged**